src/Compilers/CSharp/Portable/Symbols/TypeSymbol.cs

```diff
@@ -138,6 +138,27 @@ protected override sealed Symbol OriginalSymbolDefinition
             }
         }
 
+        // PROTOTYPE: Does this go in NamedTypeSymbol or TypeSymbol?
```

Comment:
If this only applies to `class` declarations, then it probably belongs on `NamedTypeSymbol`.